spec/3.md

```diff
@@ -0,0 +1,222 @@
+   assign arbitrary meaning to numbers on your screen.
+
+6. Zen versioned software MUST always use positive integers.
+
+7. When displaying a version, in order to differentiate from other versioning
+   schemes, it is required to prefix with a `z`
+
+## Why Use Zen Versioning?
+
+This is, unlike Semantic Versioning, a new and a revolutionary idea. In fact,
+you probably never did something remotely close to this before. The (biggest)
+problem here is that you are unimaginative, and formal specifications are
+boring.
+
+Zen Versioning establishes a simple and straightforward specification that
+communicates your intentions to the users of your software. Those intentions
+are, and always have been clear: number go up, software go new.
+
+As a responsiblee developer you will, of course, want to make sure that any
+package upgrade makes sense from the get go. Number go up = upgrade. The real
+world is a messy place, and the world of software development is worse.
+
+What you can do to remedy the unfortunate status quo is to follow a _sane_
+versioning specification while releasing and upgrading packages.
+
+## FAQ
+
+### How should I deal with revisions in the X for initial development phase?
+
+Start with 0. Increment by one every time you change something. Increment by one
+whenever you feel like. Your software will never be final, and the version
+should reflect that.
+
+### How do I know when to release 1?
+
+If your software exists, then it's good to go. Release it, what are you waiting
+for?
+
+### Doesn't this discourage rapid development and fast iteration?
+
+No.
+
+### If even the tiniest backward incompatible changes to the public API require version bump, won't I end up at version 42 very rapidly?
+
+You will, and that is a good thing. Larger number means better software, and you
+will almost certainly never run out of numbers to assign to your software.
+
+### Documenting the entire public API is too much work!
+
+Then don't, nobody cares.
+
+### What do I do if I accidentally release a backward incompatible change?
+
+Release a new version.
+
+### What should I do if I update my own dependencies without changing the public API?
+
+Increment version by one. Two if you feel brave that day, or alternatively bump
+the same number as the dependencies you have updated.
+
+### How should I handle deprecating functionality?
+
+If version of a release is below latest version, then that version is
+deprecated. Here is a useful illustration for the less mathematically inclined:
+
+```
+Latest version: 15
+Your version: 13
+```
+
+This means your version is deprecated. That is because 15 is larger than 13.
+
+### Does ZenVer have a size limit on the version string?
+
+Nope. You can have an 12 billion character version for all I care. If you find
+yourself running out of numbers, then consider that your software may be too
+good.
+
+### Is `v1` a valid zen version?
+
+No, because in ZenVer it is requires to prefix versions with a `z`
```

Comment:
This implies `z` prefix is mandatory, but it is not. z1 (or previously v1) is just as vald as `1`.

Reply:
Yes, let me clarify this